bin/utils/db_diff.py

```diff
@@ -137,30 +126,42 @@ def main(dbname, verbose, apgdiff):
     Since this script uses the command-line PostgreSQL tools any other
     configuration should be done using the various environment variables like
     PGHOST, PGPORT and PGUSER) and your `.pgpass` file.
-
-    apgdiff needs to be installed. If `apgdiff` is in your PATH it will be
-    used; otherwise you need to use `--apgdiff` or the `APGDIFF` env var to
-    specify the path to `apgdiff` apgdiff or its .jar file.
     """
     temp_dbname = 'indico_dbdiff'
-    apgdiff_cmd = _get_apgdiff_cmd(apgdiff)
-    if not apgdiff_cmd:
-        raise click.exceptions.UsageError('Could not find apgdiff in PATH; specify the path to a script or the .jar '
-                                          'file manually.')
+    base_conn = None
+    target_conn = None
+    diff = ""
```

Comment:
We are mixing `"` with `'` :P (sorry for nit-picking)

Reply:
Thanks! It's not nitpicking, it's a violation of our quotes policy 😉